.github/workflows/release_to_aur.yml

```diff
@@ -14,8 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Publish AUR package
-        uses: ATiltedTree/create-aur-release@v1
+        uses: aksh1618/update-aur-package@v1.0.5
         with:
+          tag_version_prefix: v
```

Comment:
Looks like `v` is the default value for this parameter:

> The prefix of the version used in the tag. (v by default).

I guess this is the reason why it was not needed before 🤔

Reply:
I think the fork added this parameter. v is marked as default but the parameter is marked as required.